packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html

```diff
@@ -21,7 +21,7 @@
                             data-ng-class="{
                                     'oui-status_success' : $ctrl.cluster.status === $ctrl.STATUS.READY,
                                     'oui-status_warning' : $ctrl.Kubernetes.constructor.isProcessing($ctrl.cluster.status),
-                                    'oui-status_error' : $ctrl.cluster.status === $ctrl.STATUS.ERROR
+                                    'oui-status_error' : $ctrl.Kubernetes.constructor.isError($ctrl.cluster.status) || $ctrl.cluster.status === $ctrl.STATUS.SUSPENDED,
```

Comment:
Why not use the same method as for `ERROR`status with a `isSuspended($ctrl.cluster.status)`